airflow/contrib/operators/ssh_operator.py

```diff
@@ -48,6 +48,9 @@ class SSHOperator(BaseOperator):
     :param environment: a dict of shell environment variables. Note that the
         server will reject them silently if `AcceptEnv` is not set in SSH config.
     :type environment: dict
+    :param get_pty: request a pseudo-terminal from the server. When `get_pty`
+        is enabled, the remote process will be killed upon task timeout.
+    :type get_pty: bool
```

Comment:
It would be great to mention in the comment that get_pty is forced to true when the command starts with 'sudo'